api/fixtures/example.go

```diff
@@ -111,21 +109,19 @@ func (o ExampleOptions) Resources() *ExampleResources {
 				Name:      name,
 			},
 			Data: map[string][]byte{
-				"credentials": []byte(fmt.Sprintf(`[default]
-aws_access_key_id = %s
-aws_secret_access_key = %s
-`, id, key)),
+				"config": []byte(fmt.Sprintf(`[default]
```

Comment:
why does this need to be created out of band now? wouldn't this be created and managed by our controller